example-hilt/src/main/java/app/futured/arkitekt/examplehilt/ui/base/BaseHiltActivity.kt

```diff
@@ -0,0 +1,66 @@
+package app.futured.arkitekt.examplehilt.ui.base
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.ViewDataBinding
+import androidx.fragment.app.FragmentActivity
+import app.futured.arkitekt.core.BaseViewModel
+import app.futured.arkitekt.core.ViewState
+import app.futured.arkitekt.core.event.Event
+import app.futured.arkitekt.sample.hilt.BR
+import kotlin.reflect.KClass
+
+abstract class BaseHiltActivity<VM : BaseViewModel<VS>, VS : ViewState, B : ViewDataBinding> : AppCompatActivity() {
```

Comment:
I think we should refactor BaseBindingViewModelActivity to offer a variant compatible with the Hilt so you don't have to create this class every time you use Hilt. I have created a separate issue.

https://github.com/futuredapp/arkitekt/issues/145